repair/row_level.hh

```diff
@@ -112,6 +112,21 @@ class repair_service : public seastar::peering_sharded_service<repair_service> {
     shared_ptr<row_level_repair_gossip_helper> _gossip_helper;
     bool _stopped = false;
 
+    // Possible states in which the repair service can be found.
+    //
+    // none: started, but not yet enabled. Once the repair service moves out of "none", it can
+    //       never legally move back
+    // stopped: stop() was called. The repair service will never be enabled or disabled again
+    //          and can no longer be used
+    // running: running, started and enabled at least once. Whether new compactions are accepted or not is determined by the counter
+    enum class state { none, stopped, running };
+    // The repair service is initiated in the none state. It is moved to the running state when start() is invoked
+    // and the service is immediately enabled.
+    state _state = state::none;
+    uint32_t _disabled_state_count = 0;
+
+    bool is_disabled() const { return _state != state::running || _disabled_state_count > 0; }
```

Comment:
Same can be defined in compaction_manager

Reply:
Yes, we can.